neo/core/event_BACKUP_4347.py

```diff
@@ -0,0 +1,206 @@
+# -*- coding: utf-8 -*-
```

Comment:
This files should not be here - there are other similar files generated (I think) by Git merging or rebase operations, which have been added to the commit and need to be removed.